petastorm/etl/dataset_metadata.py

```diff
@@ -89,15 +89,15 @@ def materialize_dataset(spark, dataset_url, schema, row_group_size_mb=None, use_
       indexing method. The custom indexing method is more scalable for very large datasets.
     :param pyarrow_filesystem: A pyarrow filesystem object to be used when saving Petastorm specific metadata to the
       Parquet store.
-
+    :param user: String denoting username when connecting to HDFS
```

Comment:
Maybe mention a word about the default behavior (defined by the driver used)

Reply:
done